articles/user-guide/libraries/standard/index.md

```diff
@@ -3,18 +3,31 @@ title: Introduction to Microsoft Q# standard libraries
 description: Learn about the Microsoft Q# standard libraries that define the operations, functions and data types used in quantum programs. 
 author: QuantumWriter
 ms.author: martinro
-ms.date: 12/11/2017
+ms.date: 9/24/2020
 ms.topic: article
 uid: microsoft.quantum.libraries.standard.intro
+ms.prod: quantum
 no-loc: ['Q#', '$$v']
 ---
 
-# Introduction to the Q# Standard Libraries
+# Introduction to the Q# standard libraries
 
 Q# is supported by a range of different useful operations, functions, and user-defined types that comprise the Q# *standard libraries*.
-The [`Microsoft.Quantum.Development.Kit` NuGet package](https://www.nuget.org/packages/microsoft.quantum.development.kit) installed during [installation and validation](xref:microsoft.quantum.install) provides the Q# compiler, and parts of the standard library that are implemented by the target machines.
-The [`Microsoft.Quantum.Standard` package](https://www.nuget.org/packages/microsoft.quantum.standard) provides the portion of the Q# standard libraries that are portable across target machines.
+The [`Microsoft.Quantum.Development.Kit` NuGet package](https://www.nuget.org/packages/microsoft.quantum.development.kit), installed during [installation and validation](xref:microsoft.quantum.install), provides the Q# compiler and the parts of the standard library that are implemented by the [quantum simulators](xref:microsoft.quantum.machines), or target machines.
```

Comment:
I think this package has since been deprecated, such that users should only depend on `Microsoft.Quantum.Sdk` moving forward. That package pulls in `Microsoft.Quantum.Standard` such that the next line may also not be a user-facing detail any more.